buildchain/buildchain/config.py

```diff
@@ -52,10 +52,21 @@
 class ExtCommand(enum.Enum):
     """External commands used by the build chain."""
 
-    GIT      = os.getenv('GIT_BIN',      'git')
-    HARDLINK = os.getenv('HARDLINK_BIN', 'hardlink')
-    MKISOFS  = os.getenv('MKISOFS_BIN',  'mkisofs')
-    SKOPEO   = os.getenv('SKOPEO_BIN',   'skopeo')
-    VAGRANT  = os.getenv('VAGRANT_BIN',  'vagrant')
+    GIT          = os.getenv('GIT_BIN',          'git')
+    HARDLINK     = os.getenv('HARDLINK_BIN',     'hardlink')
+    MKISOFS      = os.getenv('MKISOFS_BIN',      'mkisofs')
+    SKOPEO       = os.getenv('SKOPEO_BIN',       'skopeo')
+    VAGRANT      = os.getenv('VAGRANT_BIN',      'vagrant')
+    OPERATOR_SDK = os.getenv('OPERATOR_SDK_BIN', 'operator-sdk')
+    GOFMT        = os.getenv('GOFMT_BIN',        'gofmt')
+
+    @property
+    def command_name(self) -> str:
+        """Return the name of the command."""
+        if self is self.OPERATOR_SDK:
```

Comment:
The problem is that some command does contains `_` in their name and that would break them.

True, we aren't using such command yet, but if one day we add one, I don't want the poor guy to go looking all over the buildchain trying to understand why the command he added doesn't work.